supertokens_python/recipe/session/utils.py

```diff
@@ -418,4 +463,103 @@ def validate_and_normalise_user_input(
         app_info.framework,
         app_info.mode,
         jwt,
+        invalid_claim_status_code if (invalid_claim_status_code is not None) else 403
+        # TODO: above line was marked as TODO in review, not sure why.
```

Comment:
See how `session_expired_status_code ` is being normalised and normalise `invalid_claim_status_code` in a similar way. Also you need to check (post normalisation) if session_expired_status_code == invalid_claim_status_code, then you throw an error

Reply:
Done.